src/api/cpp/cvc5_parser.cpp

```diff
@@ -305,6 +305,9 @@ void InputParser::setIncrementalStringInput(modes::InputLanguage lang,
   // initialize the parser
   d_parser = Parser::mkParser(lang, d_solver, d_sm->toSymManager());
   initializeInternal();
+  // Ensure we've initialized the input to empty, which ensures we return
+  // null command/term prior to appendIncrementalStringInput.
```

Comment:
This comment is hard to understand without context.

Reply:
Updated